targets/TARGET_RENESAS/TARGET_VK_RZ_A1H/gpio_irq_api.c

```diff
@@ -71,6 +71,7 @@ static const PinMap PinMap_IRQ[] = {
 #else
 static const PinMap PinMap_IRQ[] = {
     {P9_1,  IRQ0, 4},
+    {P7_8,  IRQ1, 8},
```

Comment:
Basically, you pull the other commit into another branch. Due to the history here, this can get a bit complicated e.g.
```bash
# Make a new branch
git checkout origin/master -b add-rz-pins
# Move the changes from the first commit onto the branch, without making a commit
git cherry-pick -n aede40f529a7d433000dd84d9535e41e3015b2be
# Remove unrelated changes
git reset HEAD targets/targets.json tools/export/ds5_5.py tools/export/ds5_5_vk_rz_a1h.cproject.tmpl tools/export/ds5_5_vk_rz_a1h.launch.tmpl tools/export/ds5_5_vk_rz_a1h.project.tmpl  tools/export/e2studio.py tools/export/e2studio_vk_rz_a1h_cproject.tmpl  tools/export/e2studio_vk_rz_a1h_gdbinit.tmpl  tools/export/e2studio_vk_rz_a1h_project.tmpl tools/export/iar/iar_definitions.json  
# Commit these changes
git commit -m "Add missing pin to VK_RZ_A1H"
```

Or more simply
```bash
git checkout origin/master -b add-rz-pins
# Now manually add the changes (the single line addition)
git add  targets/TARGET_RENESAS/TARGET_VK_RZ_A1H/gpio_irq_api.c 
git commit -m "Add missing pin to VK_RZ_A1H"
```